### ISSUE TITLE
Null Array error when configuring IPT

### ISSUE THREAD

**Issue by Hailth** [Jackett/Jackett#1010]
Jackett version:  0.7.860.0
Mono version (if not using Windows):  Stable 4.6.2.7/08fd525
[jackettError.txt](https://github.com/Jackett/Jackett/files/740993/jackettError.txt)

-------------------------------

Hey all,

I have a weird issue when I'm trying to set up IPT, hoping this one is new since I didn't see anything like it.

I'm connecting to Jackett on a remote server using an SSH tunnel so that I can load it as a local host in my local machine's browser, and use your built in Captcha support for IPT. I'm trying to use this method as a result of not being able to get it to take the cookie headers I was passing from my local machine.

I'm able to enter in the user/pass and respond to the Captcha, but as it tries to save the settings it throws an error: Jackett.Utils.Clients.WebClientByteResult ---> System.ArgumentNullException: Array cannot be null.

What seems to be odd is that in other places of the main log it's referred to the top 10 listing on their site, as if it worked. However the setting is not confirmed on the main Jackett page so I don't think it's usable.

Any ideas?

**Comment by kaso17** [comment]
Please update to the latest jackett version and try running mono with --debug (mono --debug JacketConsole.exe) mode so we get line numbers in the exception.
It's working fine for me using cookie header.
Most likely your SSH tunnel setup doesn't work with re-captchas (does it work on other re-captcha trackers). I'll try it later myself. Are you using a single port forward or full socks proxy mode? single port forward most likely won't work. socks proxy might.

**Comment by Hailth** [comment]
Thanks for the help kaso.

I updated to the latest version and accessed the site outside the SSH tunnel (although it was running at the time) and when I went to add the tracker, it didn't prompt me for a cookie or Captcha. I figured this was something I had seen yesterday and it was going to prompt me for one or the other after it failed. But it went through just fine on its own!

Sorry I didn't realize I was on an outdated version, first time user and all. I think this can be closed.

Thanks again!
